src/pages/fieldManagement/settings/FieldTypeSelector.jsx

```diff
@@ -31,7 +31,6 @@ const fieldTypeCategories = [
       fieldTypes.file,
       fieldTypes.individual,
       fieldTypes.latlong,
-      fieldTypes.relationships,
```

Comment:
I don't see where the `FieldTypeSelector` is used. Can the entire file be deleted?

Reply:
yes